app/models/json_builders/manual_payout_report_json_builder.rb

```diff
@@ -12,7 +12,7 @@ def build
           "altcurrency" => "BAT",
           "probi" => potential_payment.amount.to_s,
           "fees" => potential_payment.fees.to_s,
-          "authority" => Publisher.find(potential_payment.finalized_by_id).to_s,
+          "authority" => Publisher.find(potential_payment.finalized_by_id).email,
```

Comment:
Could it be possible that the email could be nil?

Reply:
I  would hope not, or we have a bigger problem than a nil error 😛 

The e-mail belongs to the admin that finalized the invoice, so it has to exist before it reaches this code.